src/pages/sns.tsx

```diff
@@ -0,0 +1,480 @@
+        />
+      </Head>
+      <main className="text-black relative overflow-hidden">
+        <DarkHeroStyles></DarkHeroStyles>
+        <AnimateSpawn variants={transitions.container}>
+          <section className="overflow-hidden bg-infinite text-white">
+            <div className="container-10 pt-12 mb-60 md:mb-52 md:pt-36 relative">
+              <div className="md:w-7/10">
+                <motion.h1
+                  className="tw-heading-3 md:tw-heading-2 mb-6"
+                  variants={transitions.item}
+                >
+                  Service Nervous Systems (SNS)
+                </motion.h1>
+                <motion.p
+                  className="tw-lead-sm md:tw-lead mb-8"
+                  variants={transitions.item}
+                >
+                  Put the autonomy in DAO. No centralized power, no
+                  administrative overhead, no legal headache, only code and its
+                  community.
+                </motion.p>
+              </div>
+            </div>
+            <div className="container-10 relative">
+              <motion.img
+                src="/img/whiteBlurredCircle.png"
+                alt=""
+                className="absolute pointer-events-none max-w-none w-[800px] aspect-square -right-[200px] bottom-[-300px] md:w-[1500px] md:bottom-[-680px] md:right-[-550px] object-contain object-center"
+                variants={transitions.item}
+              />
+            </div>
+          </section>
+          <motion.section
+            className="container-12 relative h-40 sm:h-0"
+            variants={transitions.fadeIn}
+          >
+            <div className="max-w-[660px] sm:absolute pointer-events-none right-5 -translate-y-[187px] sm:-translate-y-[279px] md:-translate-y-[382px] text-center">
+              <img
+                src="/img/sns/hero.png"
+                className="w-80 sm:w-[480px] md:w-auto max-w-full"
+              />
+            </div>
+          </motion.section>
+        </AnimateSpawn>
+        <section className="container-10 sm:pt-56">
+          <AnimateSpawn
+            className="mb-20 md:mb-30"
+            variants={transitions.container}
+          >
+            <motion.p
+              className="tw-heading-5 sm:tw-heading-4 md:tw-heading-3 mb-8 md:mb-12 md:w-8/10"
+              variants={transitions.item}
+            >
+              Tired of being the product? Tired of being deplatformed by
+              corporate terms of service?
+              <br className="hidden md:inline" />
+              Be unstoppable with an SNS.
+            </motion.p>
+            <motion.p
+              className="mb-0 flex flex-col items-start sm:flex-row gap-6 md:gap-8"
+              variants={transitions.item}
+            >
+              <Link
+                className="button-outline"
+                href="/docs/current/developer-docs/integrations/sns/sns-intro#how-to-get-and-maintain-a-dao---different-options"
```

Comment:
I thinks this should point to https://internetcomputer.org/docs/current/tokenomics/sns/sns-intro-tokens 
Note that the currently linked page is not yet public

Reply:
Done. Please create PR to change the link when the docs are live.